### DISCUSSION TITLE
Can't get redgifs images IN REDDIT to blur

### DISCUSSION THREAD

**Discussion by jbsegal** [silveredgold/beta-protection#36] [Q&A]
If I'm going through reddit, almost everything from redgifs will display un-blurred.
If I open the same image on redgifs directly it blurs as expected.

redgifs.com is in my forced domains list.

I've had this same behaviour in Vivaldi and in Chrome (both on Mac).

If anyone has any ideas, I'd love to hear them...

**Comment by silveredgold** [comment]
Is Reddit enabled for censoring? The forced list (and therefore whether a page is censored) is controlled by the domain of the current page, not the source of individual images/videos.

**Reply by Wombatant1** [reply]
I don't know about op's, but i have come around the same situation - both reddit and redgifs are on my forced list, every image and video on both sites gets censord, but when it comes to redgifs vid posted on reddit, nothing happens.

win 10, latest chrome, using beta censoring as backend
